goblin_test.go

```diff
@@ -25,6 +28,73 @@ func TestRoundTripFloat(t *testing.T) {
 	}
 }
 
+type Fixture struct {
+	name string
+	goPath string
+	jsonPath string
+}
+
+func TestPackageFixtures(t *testing.T) {
+	fixtures := []Fixture {
+		Fixture{"helloworld",
+			"fixtures/packages/helloworld/helloworld.go",
+			"fixtures/packages/helloworld/helloworld.json",
+		},
+	}
+
+	for _, fix := range fixtures {
```

Comment:
Not particularly—the innards of the loop are somewhat different. I could pull out a `forEachFixture` function but that seems like more typing for not particularly more benefit.